src/api/call.ts

```diff
@@ -39,6 +39,17 @@ export const get = (path: string, data: object = {}) => {
   );
 };
 
+export const checkStatus = (response: Response) => {
+  const { status } = response;
+  if (status >= 200 && status < 300) {
+    return response;
+  } else if (status >= 500) {
+    throw new Error('Bad response from server');
```

Comment:
Maybe we could use `APIStatusCodeError` ?

Reply:
Sure, I updated `APIStatusCodeError` to use statusText as well.